pkgs/os-specific/darwin/apple-source-releases/bootstrap_cmds/default.nix

```diff
@@ -1,29 +1,18 @@
-{ lib, appleDerivation, yacc, flex }:
+{ lib, appleDerivation, pname, version, yacc, flex }:
 
 appleDerivation {
   nativeBuildInputs = [ yacc flex ];
 
+  sourceRoot="${pname}-${version}/migcom.tproj";
```

Comment:
Passing `pname` and `version` via scope is somewhat non-standard in nixpkgs. I don't understand why we need to go to such lengths just to not have `cd migcom.tproj`.